app/models/HelpCategory.scala

```diff
@@ -25,6 +25,8 @@ object HelpCategory {
   case object Epaye extends WithName("epaye") with HelpCategory
   case object SelfAssessment extends WithName("self-assessment") with HelpCategory
   case object VAT extends WithName("vat") with HelpCategory
+  case object GEN extends WithName("general") with HelpCategory
```

Comment:
Do we need to update the override of the function "to" to handle the type "GEN"?

Reply:
Yes good spot pushed up the change. had to make a change to the URL aswell. Dont merge just yet though im waiting for confirmation from Damien